### ISSUE TITLE
About The Train 

### ISSUE THREAD

**Issue by Albertchamberlain** [ljy0ustc/LLaRA#4]
![image](https://github.com/ljy0ustc/LLaRA/assets/47343901/4bd566e6-5dc8-4f80-8b19-c60e26b6c414)

Will you help me  to solve this problem ? 👆

**Comment by Albertchamberlain** [comment]
Waiting for the checkpoints😀

**Comment by ljy0ustc** [comment]
Hi~ Thanks for your attention to our work.
The termination of the process doesn't seem to be due to a lack of checkpoints, as the training process doesn't require LLaRA checkpoints, but only needs the Hugging Face model of LLaMA2-7B. Could you provide more information about your configuration of the model path for the LLaMA-2 7B Hugging Face version?

**Comment by Albertchamberlain** [comment]
> Hi~ Thanks for your attention to our work.嗨~感谢您对我们工作的关注。 The termination of the process doesn't seem to be due to a lack of checkpoints, as the training process doesn't require LLaRA checkpoints, but only needs the Hugging Face model of LLaMA2-7B. Could you provide more information about your configuration of the model path for the LLaMA-2 7B Hugging Face version?该过程的终止似乎不是由于缺乏检查点，因为训练过程不需要LLaRA检查点，只需要LLaMA2-7B的Hugging Face模型。您能提供有关LLaMA-2 7B Hugging Face版本模型路径配置的更多信息吗？

Thanks 
![image](https://github.com/ljy0ustc/LLaRA/assets/47343901/a1255353-44fa-45c7-a0cf-a938458d602f)

**Comment by Albertchamberlain** [comment]
It's seem‘s like no problem about the  LLaMA-2 7B Hugging Face config path or version
It‘s just be killed😂

**Comment by ljy0ustc** [comment]
It seems your program was killed while executing the line of code ```self.llama_model = LlamaForCausalLM.from_pretrained(llm_path, torch_dtype=torch.bfloat16)``` in mode/model_interface.py. Perhaps you could try directly loading the LLaMA-2 7B Hugging Face version model under your conda environment and see if it can be loaded normally.

**Comment by Albertchamberlain** [comment]
> It seems your program was killed while executing the line of code `self.llama_model = LlamaForCausalLM.from_pretrained(llm_path, torch_dtype=torch.bfloat16)` in mode/model_interface.py. Perhaps you could try directly loading the LLaMA-2 7B Hugging Face version model under your conda environment and see if it can be loaded normally.似乎您的程序在执行 mode/model_interface.py 中的代码行 `self.llama_model = LlamaForCausalLM.from_pretrained(llm_path, torch_dtype=torch.bfloat16)` 时被终止。也许您可以尝试在您的 conda 环境下直接加载 LLaMA-2 7B Hugging Face 版本模型，看看是否可以正常加载。

Ok，got it，i will try later，Thx a lot ！！！

**Comment by Albertchamberlain** [comment]
Yeah, as you said, I reset the LLaMA-2 7B Path in another machine.   It's work
![image](https://github.com/ljy0ustc/LLaRA/assets/47343901/fc05d176-1dfe-4bca-8026-92e72f005991)

**Comment by Albertchamberlain** [comment]
Hello,another Q，How can i get this file？

![image](https://github.com/ljy0ustc/LLaRA/assets/47343901/3c7d567e-d1c3-4984-b921-5053b5ee5921)

I have download the dataset 
![image](https://github.com/ljy0ustc/LLaRA/assets/47343901/4c943522-d840-4109-ba3f-17d3ba217496)

**Comment by lavinal712** [comment]
> Yeah, as you said, I reset the LLaMA-2 7B Path in another machine. It's work ![image](https://private-user-images.githubusercontent.com/47343901/313098143-fc05d176-1dfe-4bca-8026-92e72f005991.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI1NTk5NDUsIm5iZiI6MTcxMjU1OTY0NSwicGF0aCI6Ii80NzM0MzkwMS8zMTMwOTgxNDMtZmMwNWQxNzYtMWRmZS00YmNhLTgwMjYtOTJlNzJmMDA1OTkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDA3MDA0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwZTgxZjlkMDdmODQ1MDg2N2M5NjY1ZjVlYmY2NTJjYjZjMWNlMGEzM2VlNmJjYmM2ZTRiYTkyNGNjOGZlMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6yacVgfqR6MmQyfJLwB8bWik71jQxOOZsr2LPYX7vxE)

Hello, I meet some problem about this code. Did you run the code successfully?
